src/backend/tests/c-lisp/kernel/MMult1.sexp

```diff
@@ -0,0 +1,39 @@
+(c-lisp
+
+    (define ((fprint float) (n float)))
+    (define ((print int) (n int)))
+
+    (define ((__MMult1 void) (A (ptr float))
+                            (B (ptr float))    
+                            (C (ptr float))    
+                            (m int)    
+                            (n int)    
+                            (k int)    
+            )
```

Comment:
can you look at formatting we did in other tests and format this to that? we don't keep brackets tangling this this as in C